### DISCUSSION TITLE
3D-printed case for FabGL ESP32 board (3.3v)?

### DISCUSSION THREAD

**Discussion by vitasam** [fdivitto/FabGL#300] [Q&A]
Hello,
I received the FabGL board today, the firmware upload was successful - PC Emulator works out of the box (however, the pull request is required). Great board! 

I have a question: Is there a 3D model for enclosure (case) for this board?
![fabgl_board_3 3v](https://user-images.githubusercontent.com/5597505/200666367-ec36ba4c-2c3d-4f59-8290-fbf1f34573ea.jpg)

**Comment by johngalt01** [comment]
i couldn't find any.

there are a few versions for the VGA32 V1.1-1.4. it wouldn't be to hard to design a case just to enclose it but everyone has different needs and different configurations with things plugged into the i/o pins.

**Reply by vitasam** [reply]
I see. I have quite nice 3D model for TTGO VGA32 V1.4, from [Thingverse](https://www.thingiverse.com/thing:4565213). There is also quite interesting [case](https://www.thingiverse.com/thing:5028782) available.

Maybe I will start to learn FreeCAD again.

**Reply by johngalt01** [reply]
i'm using this one [https://www.thingiverse.com/thing:4790919](url)
it has room for a 232 9 pin converter makes for a compact terminal or pc emulator.

**Reply by vitasam** [reply]
Yes, I saw this case on Thingverse site. As far as I understand, it is more suitable for Serial Terminal board rather than for development board.
